### ISSUE TITLE
Bypass "workspace select" if the workspace state is already the desired one

### ISSUE THREAD

**Issue by LiuVII** [minamijoyo/tfmigrate#46]
Hey!

In our organization, we're using Scalr https://docs.scalr.com/en/latest/ to perform terraform CI/CD actions.

One of the limitations we're facing to adopt `tfmigrate` is that we can't use `terraform workspace select` command because remote workspaces are configured as

```
terraform {
  backend "remote" {
    hostname = "<account>.scalr.io"
    organization = "<org-id>"

    workspaces {
      name = "<workspace-name>"
    }
  }
}
```

But still `terraform workspace show` is available and all other commands that are not related to workspace operations.

So I wonder if it's possible/reasonable to add a simple logical check:  if selected workspace is already the one that's required by `tfmigrate` configuration (using `terraform workspace show` maybe?) then bypass `terraform workspace select` command 🙏 

Upd: actually, the issue is not with Scalr, it's core terraform code, workspace select is only available for `workspaces.prefix` which is not how we'd like to configure our remote

**Comment by minamijoyo** [comment]
Hi, @LiuVII. Thank you for opening the issue. It sounds reasonable for me.

**Comment by LiuVII** [comment]
Awesome! So swift as well!
@minamijoyo lmk if I can assist in any way to get this merged and released 🙂

**Comment by minamijoyo** [comment]
@LiuVII Fixed in v0.2.10 🚀
